api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -19,12 +19,12 @@
 
 
 def build_preprint_update_payload(
-        node_id,
-        attributes=None,
-        relationships=None):
+        node_id, attributes=None, relationships=None,
+        content_type='preprints'):
```

Comment:
`content_type` is easily confused with `ContentType` in Django or `Content-Type` in HTTP. How about `jsonapi_type`?